packages/react-core/src/components/Form/examples/Form.md

```diff
@@ -770,3 +781,433 @@ class SimpleForm extends React.Component {
+      value8: '',
+      value9: '',
+      value10: '',
+      value11: '',
+      value12: '',
+      value13: '',
+      value14: '',
+      value15: '',
+      value16: '',
+      value17: '',
+      value18: '',
+      value19: '',
+      value20: '',
+      value21: '',
+      value22: ''
+    };
+    this.handleTextInputChange = value => {
+      this.setState(value);
+    };
+  }
+
+  render() {
+    const {
+      value1,
+      value2,
+      value3,
+      value4,
+      value5,
+      value6,
+      value7,
+      value8,
+      value9,
+      value10,
+      value11,
+      value12,
+      value13,
+      value14,
+      value15,
+      value16,
+      value17,
+      value18,
+      value19,
+      value20,
+      value21,
+      value22
+    } = this.state;
+
+    return (
+      <Form>
+        <FormGroup label="Label 1" isRequired fieldId="form-expandable-field-groups-label1">
+          <TextInput
+            isRequired
+            id="form-expandable-field-groups-label1"
+            name="form-expandable-field-groups-label1"
+            value={value1}
+            onChange={value => {
+              this.handleTextInputChange({ value1: value });
+            }}
+          />
+        </FormGroup>
+        <FormGroup label="Label 2" isRequired fieldId="form-expandable-field-groups-label2 ">
+          <TextInput
+            isRequired
+            id="form-expandable-field-groups-label2"
+            name="form-expandable-field-groups-label2"
+            value={value2}
+            onChange={value => {
+              this.handleTextInputChange({ value2: value });
+            }}
+          />
+        </FormGroup>
+        <FormFieldGroupExpandable
+          isExpanded
+          toggleAriaLabel="Details"
+          header={
+            <FormFieldGroupHeader
+              titleText={{text: "Field group 1", id: "field-group1-titleText-id"}}
+              titleDescription="Field group 1 description text."
+              actions={
+                <>
+                  <Button variant="link">Delete all</Button> <Button variant="secondary">Add parameter</Button>
+                </>
+              }
+            />
+          }
+        >
+          <FormFieldGroupExpandable
+            isExpanded
+            toggleAriaLabel="Details"
+            header={
+              <FormFieldGroupHeader
+                titleText={{text: "Nested field group 1", id: "nested-field-group1-titleText-id"}}
+                titleDescription="Nested field group 1 description text."
+                actions={
+                  <Button variant="plain" aria-label="Remove">
+                    <TrashIcon />
+                  </Button>
+                }
+              />
+            }
+          >
+            <FormGroup label="Label 1" isRequired fieldId="form-expandable-field-groups-field-group1-label1">
+              <TextInput
+                isRequired
+                id="form-expandable-field-groups-field-group1-label"
```

Comment:
This is where one of the issues is coming from. See how the id and name don't match up with the fieldId for the label 1? You can see how the FormGroup below it matches. I think if you get these matching (maybe "form-expandable-field-groups-field-group1-label1), it'll fix that issue.